### PR TITLE
Update sendinblue link

### PR DESCRIPTION
**Pull request by yanotoma**
#### By submitting this pull request, I promise that:

- [x] I have read the [contribution guidelines](https://github.com/strapi/awesome-strapi/blob/master/CONTRIBUTING.md) thoroughly.
- [x] I ensure my submission follows each and every point.

### CODE REVIEW

**Comment by Lith** [comment]
Hi,

I think it will be better to let the old version, I'm using the version of @eddybordi since 2 years now... and yes, he's not maintain the repository, and yes, it's a bit annoying to have my PR in review since many month (https://github.com/eddybordi/strapi-provider-email-sendinblue/pull/5/files) 

I suggest to add your version, but keep the oldest ?

Thanks !

File: README.md
Comment (derrickmehaffy) [inline README.md:168]: I concur with @Lith, can you please modify this to state the previous package as outdated and add yours below it as a new version?

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]

**Comment by Lith** [comment]
I've send a message to eddybordi, and he will give me right to npmjs repos, so I will be able to maintain it and keep it up to date.
So... I'm not sure this new package will be interesting here (too confused?). Just wait a few days :) 

Otherwise @yanotoma, we really appreciate your work, the main object here is to avoid confusion with 2 same packages.
It's better to have once with many collaborators, instead of many with one collaborator.

**Comment by derrickmehaffy** [comment]
> I've send a message to eddybordi, and he will give me right to npmjs repos, so I will be able to maintain it and keep it up to date.
> So... I'm not sure this new package will be interesting here (too confused?). Just wait a few days :)
> 
> Otherwise @yanotoma, we really appreciate your work, the main object here is to avoid confusion with 2 same packages.
> It's better to have once with many collaborators, instead of many with one collaborator.

Thank you very much @Lith I agree it would be better to have a single source of truth with multiple contributors. For the moment I'll place this PR on hold until we have a confirmed answer as to the lead maintainer for the package.

**Comment by yanotoma** [comment]
Hi guys, sorry for not answer 
I agree to have only one repo for this, so I think the current repo can stay if you can update it I have no problem with that
You can check my repo or let me know if I can help with something

**Comment by derrickmehaffy** [comment]
@Lith did you hear back from eddybordi ?

**Comment by derrickmehaffy** [comment]
Going to close this as we haven't heard any information yet. We can reopen once we have some more updated information.

**Pull request closed without merging**
